lib/stubs/object_ml.c

```diff
@@ -0,0 +1,135 @@
+#include "lablqml.h"
```

Comment:
This file is marked completely new because `git mv` was not used? Or is it really completely new?